src/main/java/fourstep/logic/Main.java

```diff
@@ -0,0 +1,28 @@
+package fourstep.logic;
+
+
+import fourstep.view.InputView;
+import fourstep.view.ResultView;
+
+import java.util.ArrayList;
+
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        int raceCount = InputView.inputRaceCount();
+        String[] carNames = InputView.inputCarName();
+
+        RandomNumber randomNumber = new RandomNumber();
+
+        Race race = new Race(carNames, raceCount);
+
+        ArrayList<String> winners = race.play(randomNumber);
```

Comment:
```suggestion
        List<String> winners = race.play(randomNumber);
```

수정된 코드는 기존 코드와 어떤 차이가 있을까요?